src/api/m64p_types.h

```diff
@@ -224,11 +224,11 @@ typedef enum
 
 typedef enum
 {
-    SAVETYPE_EEPROM_4KB = 0,
-    SAVETYPE_EEPROM_16KB,
+    SAVETYPE_EEPROM_4K = 0,
+    SAVETYPE_EEPROM_16K,
     SAVETYPE_SRAM,
     SAVETYPE_FLASH_RAM,
-    SAVETYPE_CONTROLLER_PACK,
+    SAVETYPE_CONTROLLER_PAK,
```

Comment:
"Correct" spelling is "Pak": https://en.wikipedia.org/wiki/Nintendo_64_accessories#Controller_Pak